app/uk/gov/hmrc/transitmovementsauditing/models/Metadata.scala

```diff
@@ -37,6 +38,7 @@ object Metadata {
 
 case class Metadata(
   path: String,
+  subtype: Option[String],
```

Comment:
As suggested with `Details`, we should also make a `MetadataRequest` object, as this subtype should not be user specified.

For that matter, actually, it shouldn't be on here, if you look at the auditing page I put the subtype on the root of the details object, intentionally so as the metadata relates to the request, not the typing. So this is in the wrong place anyway -- please have a look at the confluence page I wrote.